samples/zipkin-sleuth/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.1'
+services:
+  zipkin:
+    image: openzipkin/zipkin:latest
+    ports:
+      - "9411:9411"
+  sleuth-zipkin:
+    image: sleuth-zipkin:0.0.1-SNAPSHOT
```

Comment:
@sdeleuze it should be `zipkin-sleuth` here. My bad. Will you fix it directly in main?

Reply:
Sure